src/families/filecoin/test-dataset.ts

```diff
@@ -0,0 +1,219 @@
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 100,
+                gasPremium: "200",
+                recipient: "novalidaddress",
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  recipient: new InvalidAddress(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Fee Required (Gas Premium)",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 100,
+                gasPremium: "0",
+                recipient: ACCOUNT_1,
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  gas: new FeeNotLoaded(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Fee Required (Gas Fee Cap)",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "0",
+                gasLimit: 100,
+                gasPremium: "10000",
+                recipient: ACCOUNT_1,
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  gas: new FeeNotLoaded(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Fee Required (Gas Limit)",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 0,
+                gasPremium: "10000",
+                recipient: ACCOUNT_1,
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  gas: new FeeNotLoaded(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Not enough balance",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 10,
+                gasPremium: "10000",
+                recipient: ACCOUNT_1,
+                amount: "10000000000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  amount: new NotEnoughBalance(),
```

Comment:
one test also seem to see a problem that the transaction you express is not > the account balance or that there are not error happening. 

you may want to check if this happens when you do:

```
ledger-live getTransactionStatus --id filecoin:t1d2xrzcslx7xlbbylc5c3d5lvandqw4iwl6epxba --amount 100000 --recipient t1d2xrzcslx7xlbbylc5c3d5lvandqw4iwl6epxba
```